Author: Filip Hroch <hroch@physics.muni.cz>
Description: NULL dereference bug
--- a/dcraw.c
+++ b/dcraw.c
@@ -284,8 +284,10 @@
 {
   if (!data_error) {
     fprintf (stderr, "%s: ", ifname);
-    if (feof(ifp))
+    if (feof(ifp)) {
       fprintf (stderr,_("Unexpected end of file\n"));
+      longjmp (failure, 1);
+    }
     else
       fprintf (stderr,_("Corrupt data near 0x%llx\n"), (INT64) ftello(ifp));
   }
