Index: alsa-utils-1.2.3/alsactl/alsa-restore.service.in
===================================================================
--- alsa-utils-1.2.3.orig/alsactl/alsa-restore.service.in
+++ alsa-utils-1.2.3/alsactl/alsa-restore.service.in
@@ -13,5 +13,6 @@ After=alsa-state.service
 [Service]
 Type=oneshot
 RemainAfterExit=true
+ExecStartPre=/bin/mkdir -p @alsactlhome@
 ExecStart=-@sbindir@/alsactl -E HOME=@alsactlhome@ -E XDG_RUNTIME_DIR=@alsactlruntime@ restore
 ExecStop=-@sbindir@/alsactl -E HOME=@alsactlhome@ -E XDG_RUNTIME_DIR=@alsactlruntime@ store
Index: alsa-utils-1.2.3/alsactl/alsa-state.service.in
===================================================================
--- alsa-utils-1.2.3.orig/alsactl/alsa-state.service.in
+++ alsa-utils-1.2.3/alsactl/alsa-state.service.in
@@ -10,5 +10,6 @@ ConditionPathExists=@daemonswitch@
 
 [Service]
 Type=simple
+ExecStartPre=/bin/mkdir -p @alsactlhome@
 ExecStart=-@sbindir@/alsactl -E HOME=@alsactlhome@ -E XDG_RUNTIME_DIR=@alsactlruntime@ -s -n 19 -c rdaemon
 ExecStop=-@sbindir@/alsactl -E HOME=@alsactlhome@ -E XDG_RUNTIME_DIR=@alsactlruntime@ -s kill save_and_quit
